theme-check.rb

```diff
@@ -0,0 +1,109 @@
+# -*- ruby -*-
+# autogenerated via `brew gem formula theme-check`
+# https://github.com/sportngin/brew-gem
+
+require 'formula'
+require 'fileutils'
+
+USE_HOMEBREW_RUBY = true
+
+module RubyBin
```

Comment:
I would move this module and the class bellow nested under the `class ThemeCheck`. Since the file is named `theme_check.rb`, best practice is to only add a constant of the same name to the global namespace (so just `ThemeCheck`.

Reply:
:+1: